tests/utils/splitAtom.test.tsx

```diff
@@ -3,7 +3,7 @@ import type { ChangeEvent } from 'react'
 import { fireEvent, render, waitFor } from '@testing-library/react'
 import { atom, useAtom } from 'jotai'
 import type { Atom, PrimitiveAtom } from 'jotai'
-import { splitAtom } from 'jotai/utils'
+import { splitAtom, useAtomValue, useUpdateAtom } from 'jotai/utils'
```

Comment:
I would like to simply use `useAtom` only, if that's not a problem to reproduce the test.